### PR TITLE
Add cloudcompiler.yaml

### PR DESCRIPTION
**Pull request by fschutt**
Adds a GitHub workflow to publish wasmer.wasm to wapm.dev

- [x] Add a short description of the change to the CHANGELOG.md file

### CODE REVIEW

File: .github/workflows/cloudcompiler.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      - name: Install wasm32-wasi target
+        shell: bash
+        run: |
+          rustup target add wasm32-wasi
+      - name: Build cloudcompiler.wasm
+        shell: bash
+        run: |
+          cargo build --release --target wasm32-wasi &&
+          mkdir target/release/cloudcompiler &&
+          cp target/release/wasmer.wasm target/release/cloudcompiler/cloudcompiler.wasm &&
```
Comment (ptitSeb) [inline .github/workflows/cloudcompiler.yaml:32]: For me, the file is generated in `target/wasi32-wasm/release` and is name `wasmer-compiler.wasm`

Comment (fschutt) [reply .github/workflows/cloudcompiler.yaml:32]: yeah this was just a test / initial commit, I need to rework this script anyway

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -191,6 +191,11 @@ jobs:
         shell: bash
         run: |
           make build-wasmer
+      - name: Build Wasmer binary on Wasm32-WASI without LLVM
+        if: matrix.use_llvm != true
```
Comment (syrusakbary) [inline .github/workflows/build.yml:195]: I'm not sure this is the right flag, we may need a new flag which is `build_wasm`

**Review by syrusakbary — APPROVED** [review]

**Comment by syrusakbary** [comment]
Merging manually as no steps have changed other than the build wasm

**Pull request merged**